br refactor (2): translate br overview, design overview, and use overview (#11313)
pingcap/docs: br/br-use-overview.md

* Which backup storage system should I choose?
* In which directory should I place the backup data during a backup task?
* In what way should I organize the directory of the full backup data and log backup data?
* How to handle the historical backup data in the storage system?

The following sections will answer these questions one by one.

**Choose a backup storage system**

It is recommended that you store backup data to Amazon S3, Google Cloud Storage (GCS), or Azure Blob Storage. Using these systems, you do not need to worry about the backup capacity and bandwidth allocation.

If the TiDB cluster is deployed in a self-built data center, the following practices are recommended:

* Build [MinIO](https://docs.min.io/docs/minio-quickstart-guide.html) as the backup storage system, and use the S3 protocol to back up data to MinIO.
* Mount Network File System (NFS, such as NAS) disks to br command-line tool and all TiKV instances, and use the POSIX file system interface to write backup data to the corresponding NFS directory.

> **Note:**
>

> > If no NFS is mounted to the br command-line tool or TiKV node, or if you use a storage system that supports Amazon S3, GCS, or Azure Blob Storage protocols, the data backed up by br command-line tool is generated at each TiKV node. **Note that this is not the recommended way to deploy the br command-line tool**, because the backup data are scattered in the local file system of each node. Collecting the backup data might result in data redundancy and operation and maintenance problems. Meanwhile, if you restore data directly before collecting the backup data, you will encounter the `SST file not found` error.

Reviewer: ```suggestion
> If you don't choose NFS or a storage system that supports Amazon S3, GCS, or Azure Blob Storage protocols, the data backed up is generated at each TiKV node. **Note that this is not the recommended way to use BR**, because collecting the backup data might result in data redundancy and operation and maintenance problems. 
```

Author: The change should be synchronized to Chinese. Otherwise, the Chinese and English are inconsistent.

**Organize the backup data directory**

* Store the snapshot backup and log backup in the same directory for unified management, for example, `backup-${cluster-id}`.

> * Store each snapshot backup in a directory with the backup date included, for example, `backup-${cluster-id}/fullbackup-202209081330`.

Reviewer: ```suggestion
* Store each snapshot backup in a cutom directory , for example, `backup-${cluster-id}/fullbackup-202209081330`.
```

> * Store the log backup in a fixed directory, for example, `backup-${cluster-id}/logbackup`. The log backup program creates a subdirectory in the `logbackup` directory every day to distinguish the data backed up each day.

Reviewer: ```suggestion
* Store the log backup in a fixed directory, for example, `backup-${cluster-id}/logbackup`. The log backup program will create subdirectories under the `logbackup` directory every day to distinguish the data backed up each day.
```

Author: We seldom ues future tense in technical documents unless there is obvious time point.

**Handle historical backup data**

> Assume that you have specified a **backup retention period**. Backup data will be retained only for this period, for example, 7 days. The concept **backup retention period** will also be mentioned in backup tutorials.

Reviewer: ```suggestion
Assume that you need to set the life cycle for each backup data, for example, 7 days. The concept **backup retention period** will also be mentioned in backup tutorials.
```

Author: This change should be synced to Chinese as well.

> * To perform PITR, you need to restore the full backup before the restore point, and the log backup between the full backup and the restore point. Therefore, **It is recommended to only delete the log backup before the full snapshot**.  For log backups that exceed the backup retention period, you can use `br log truncate` command to delete the backup before the specified time point.

Reviewer: ```suggestion
* To perform PITR, you need to restore the full backup before the restore point, and the log backup between the full backup and the restore point. Therefore, **It is recommended to only delete the log backup before the full snapshot**. For log backups that exceed the backup retention period, you can use `br log truncate` command to delete the backup before the specified time point.
```

> * To perform PITR, you need to restore the full backup before the restore point, and the log backup between the full backup and the restore point. Therefore, for log backups that exceed the backup retention period, you need to run the `br log truncate` command to delete the backup before the specified time point. **It is recommended to only delete the log backup before the full snapshot**.

Reviewer: ```suggestion
* To perform PITR, you need to restore the full backup before the restore point, and the log backup between the full backup and the restore point. Therefore, **It is recommended to only delete the log backup before the full snapshot**.  For log backups that exceed the backup retention period, you can use `br log truncate` command to delete the backup before the specified time point 
```

Author: This change should be synced to Chinese as well.

* For backup data that exceeds the retention period, you can delete or archive the backup directory.

### How to restore data?

- To restore only full backup data, you can use `br restore` to perform a full restore of the specified backup.

> - If you have started log backup and regularly performed a full backup, you can run the `br restore point` command to restore backup data to any time point within the backup retention period.

Reviewer: ```suggestion
- If you have started log backup and regularly performed a full backup, you can run the `br restore point` command to restore to any time point within the backup retention period.
```
